components/ide/jetbrains/image/status/main.go

```diff
@@ -443,16 +431,46 @@ func run(launchCtx *LaunchContext) {
 	os.Exit(cmd.ProcessState.ExitCode())
 }
 
-func remoteDevServerCmd(args []string, productContext *LaunchContext) *exec.Cmd {
-	cmd := exec.Command(BackendPath+"/bin/remote-dev-server.sh", args...)
-	cmd.Env = os.Environ()
+func remoteDevServerCmd(args []string, launchCtx *LaunchContext) *exec.Cmd {
+	if launchCtx.env == nil {
+		launchCtx.env = os.Environ()
 
-	// Set default config and system directories under /workspace to preserve between restarts
-	cmd.Env = append(cmd.Env,
-		fmt.Sprintf("IJ_HOST_CONFIG_BASE_DIR=%s", productContext.configDir),
-		fmt.Sprintf("IJ_HOST_SYSTEM_BASE_DIR=%s", productContext.systemDir),
-	)
+		// Emulate the interactive login shell to ensure that all users defined shell scripts are loaded
+		shell := os.Getenv("SHELL")
+		if shell == "" {
+			shell = "/bin/bash"
+		}
+		envCmd := exec.Command(shell, []string{"-ilc", "env"}...)
```

Comment:
it is better to have a custom go program which print os.Environ to avoid issues with escapes characters by using JSON